src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider applying the font family for the entire page to get it inherited by other elements